languages/swift/exercises/concept/closures/.docs/instructions.md

```diff
@@ -0,0 +1,111 @@
+In this exercise you will be using closures with a higher-order function to generate sequences of numbers from a seed number.
+
+You are given the function
+
+```swift
+unroll(_: Int,
+       generator: (Int) -> (seed: Int, value: Int),
+       stop: (Int) -> Bool
+      ) -> [Int]
+```
+
+`unroll` takes three parameters, a seed value; a `generator` function which, when applied to the seed returns a tuple consisting of the next value for the seed and the next value for the sequence; and a `stop` function which, when applied to the current seed value returns true if the sequence generation is finished. Unroll then returns the generated sequence. For example, to generate the powers of 3 up to 1,000,000, we can write:
+
+```swift
+let powersOf3 = unroll(1,
+                       generator: { (n: Int) -> (seed: Int, next: Int) in (seed: 3 * n, next: n) },
+                       stop: { (n: Int) -> Bool in n > 1_000_000 })
+// => [1, 3, 9, 27, 81, 243, 729, 2187, 6561, 19683, 59049, 177147, 531441]
+```
+
+You will use unroll to help you generate a few sequences.
+
+## 1. Write helper closures to generate a sequence of digits
```

Comment:
This task description describes _how_ to implement the task, instead of _what_ to implement (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd) for guidance). This also ties into the story/theme comment I made above.

Note: this comment also applies to the other tasks.